src/attributes.jl

```diff
@@ -472,9 +472,9 @@ struct Name <: AbstractModelAttribute end
 """
     ObjectiveSense()
 
-A model attribute for the `OptimizationSense` of the objective function, which
-can be `MIN_SENSE`, `MAX_SENSE`, or `FeasiblitySense`. The default is
-`FEASIBILITY_SENSE`.
+A model attribute for the objective sense of the objective function, which
+must be an `OptimizationSense`: `MIN_SENSE`, `MAX_SENSE`, or
```

Comment:
Use `[OptimizationSense](@ref)`

Reply:
`OptimizationSense` has no docstring, so this won't be useful.